### DISCUSSION TITLE
Increasing sampling steps to 250

### DISCUSSION THREAD

**Discussion by mpmo10** [hlky/stable-diffusion#266] [Ideas]
Hi, today I found your webui and i think it's superior to the hlky webui however one thing which I prefer in the hlky webui is that I can increase the sampling steps to 250, while in yours I can only increase to 150. So is it pls possible to increase it to 250? Oh and can you also increase batch count limit from 16 to 100 as well?

**Comment by slashedstar** [comment]
If you go to frontend/frontend.py you can edit these lines:

`txt2img_steps = gr.Slider(minimum=1, maximum=100, step=1, label="Sampling Steps"`
Change the maximum to whatever number you'd like so you can go from 1 to whatever steps

Bath count:
`txt2img_batch_count = gr.Slider(minimum=1, maximum=20`

You can use any text editor to search for the `txt2img_steps = gr.Slider`, `img2img_steps = gr.Slider`, `txt2img_batch_count = gr.Slider` and  `img2img_batch_count = gr.Slider` to edit the maximum value

**Reply by mpmo10** [reply]
Aha thankss

**Reply by mpmo10** [reply]
> If you go to frontend/frontend.py you can edit these lines:
> 
> `txt2img_steps = gr.Slider(minimum=1, maximum=100, step=1, label="Sampling Steps"` Change the maximum to whatever number you'd like so you can go from 1 to whatever steps
> 
> Bath count: `txt2img_batch_count = gr.Slider(minimum=1, maximum=20`
> 
> You can use any text editor to search for the `txt2img_steps = gr.Slider`, `img2img_steps = gr.Slider`, `txt2img_batch_count = gr.Slider` and `img2img_batch_count = gr.Slider` to edit the maximum value


So I found only two files called frontend.py, but I can't find "txt2img_steps = gr.Slider" in either one of them. I also can't find the "txt2img_batch_count" command anywhere in the files. I used notepad++ to search for them but nothing :( (attached shows the two files I found in the stable-diffusion-webui-master folder)

![2022-09-04 20_25_38-frontend py - Search Results in stable-diffusion-webui-master](https://user-images.githubusercontent.com/52265226/188323445-c52688cd-8a57-4183-9835-51badf5d6253.png)
